environments/map_env.yaml

```diff
@@ -0,0 +1,19 @@
+name: timelapse_map_env
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+  - pip
+  - jupyter
+  - ipykernel
+  - pre-commit
+  - black
+  - pyarrow
+  - seaborn
+  - matplotlib
+  - pytoml
+  - pip:
+    - git+https://github.com/cytomining/copairs.git@v0.5.0
+    - kaleido
+    - pycytominer
+    - plotly
```

Comment:
Is there any particular reason to install these packages via pip (except for copairs)? If not, consider adding them only in the conda dependencies section to avoid version mismatches due to the use of multiple package managers.